macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -499,9 +499,13 @@ open class AvatarView: NSView {
 		if initials == nil,
 			let email = email?.trimmingCharacters(in: whitespaceNewlineAndZeroWidthSpace),
 			!email.isEmpty {
-			let initialCharacter = email[email.startIndex]
-			if initialCharacter.isValidInitialsCharacter {
-				initials = String(initialCharacter)
+			let emailComponentsWithUnicodeLetterFirstCharacters = email.filter {
+				$0.isValidInitialsCharacter
+			}
+
+			// Find the first valid unicode character in the email address and use that
+			if let initial = emailComponentsWithUnicodeLetterFirstCharacters.first {
```

Comment:
How do we ensure this behavior is used consistently across all of our platforms (since this is outside of the hashing code)?

Reply:
I don't think we can at this point, at least not without modifying the public API to align with iOS. I agree that it's not good, but unfortunately we'd have to prioritize that against our existing iOS work.